mozregression/fetch_build_info.py

```diff
@@ -270,6 +271,7 @@ def find_build_info(self, date, fetch_txt_info=True, max_workers=2):
             for thread in threads:
                 while thread.is_alive():
                     thread.join(0.1)
+            LOG.debug("got valid_builds %s" % valid_builds)
```

Comment:
Could you remove these debugging logs as well?

Reply:
Is there rationale for when debug logs are ok vs not? I found these logs useful when debugging the code, so I figured they might be useful to others too.

Reply:
Not really. Actually, on second thought, let's leave these in. They shouldn't be visible under normal circumstances.